eclide/mainfrm.cpp

```diff
@@ -2531,11 +2537,11 @@ void CMainFrame::DoWorkspaceLoad(IWorkspace * workspace)
         }
     }
     IWorkspaceVector workspaceItems;
-    workspace->GetChildWorkspaces(workspaceItems);
-    for(IWorkspaceVector::const_iterator itr = workspaceItems.begin(); itr != workspaceItems.end(); ++itr)
-    {
-        DoWorkspaceLoad(itr->get());
-    }
+    //workspace->GetChildWorkspaces(workspaceItems);
+    //for(IWorkspaceVector::const_iterator itr = workspaceItems.begin(); itr != workspaceItems.end(); ++itr)
+    //{
+    //    DoWorkspaceLoad(itr->get());
+    //}
```

Comment:
Commented code

Reply:
Yes. I just saw that. Happened when I rebased. Will look at that.